_posts/2016-12-19-2016-year-in-review.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title:  "Apache Flink in 2016: Year in Review"
+date:   2016-12-19 09:00:00
+author: "Mike Winters"
+excerpt: "<p>As 2016 comes to a close, let's take a moment to look back on the Flink community's great work during the past year.</p>"
+categories: news
+---
+<p>2016 was an exciting year for the Apache Flink® community, and the
+  <a href="http://flink.apache.org/news/2016/03/08/release-1.0.0.html" target="_blank">release of Flink 1.0 in March</a>
+   marked the first time in Flink’s history that the community guaranteed backward compatibility for all
```

Comment:
I like the suggestion--I'll change the language to _API backward compatibility_. As for the side question: according to my sources, both are correct, but 'backward' is more common in US English and 'backwards' is more common in British English :)